hnn_core/network.py

```diff
@@ -1019,6 +1032,10 @@ def add_bursty_drive(
         conn_seed : int
             Optional initial seed for random number generator (default: 3).
             Used to randomly remove connections when probability < 1.0.
+        gid_start : int, default: None
```

Comment:
Maybe add the default values for the synaptic delays and space constants like it was done in the previous modifications? No need to overhaul all the docstrings, but just to keep consistent with what is changed in the other drives